tests/fpga_block_tests/adc_model/test_adc_model.py

```diff
@@ -25,14 +29,14 @@ def model(in_, n, vref):
 
     # determine magnitude
     in_abs = abs(in_)
-    mag_real = (abs(in_abs) / vref) * ((2**(n-1)) - 1)
+    mag_real = (abs(in_abs) / CFG['vref']) * ((2**(CFG['n']-1)) - 1)
```

Comment:
there was not a single source of truth here, so i'm trying to fix that with this change